index.js

```diff
@@ -6,9 +6,7 @@ module.exports = function () {
     return function () {};
   }
   var active = document.activeElement;
-  var ranges = Array.apply(Array, {
-    length: selection.rangeCount
-  }).map(function(range, index) {
+  var ranges = Array.apply(Array, [selection.rangeCount]).map(function(range, index) {
```

Comment:
to make it work in phantom too I suggest to use dumb but working solution: `Array(selection.rangeCount).join(' ').split(' ').map(function(range, index) {`